docs/ides/vscode.rst.include

```diff
@@ -10,13 +10,295 @@ It can be used together with any other coding language or be used to edit config
+
+**Run and debug:** is essential for debugging your code and running your applications within VSCode.
+
+**Extensions:** an *App store* equivalent for VS Code; it is your gateway to customizing and extending the functionality of VSCode through various extensions.
 VS Code is an extensible and versatile development environment.
 The extensions can greatly aid during the coding of varying problems.
 Some notable highlights include automatic API messages for methods, and
 integration with Jupyter notebooks.
 
-Extensions can be installed by following `these instructions <https://code.visualstudio.com/docs/editor/extension-marketplace>`__.
 
-Once installed it is vital to ensure the Python interpreter is pointing to the *correct* Python executable.
-This depends on whether you are using :ref:`virtual environments <python-environments>`, see :ref:`here <environment-venv-vscode>`.
-Changing the interpreter location can be done by following `this instruction <https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter>`__.
+Creating and Opening Projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Create a folder for a project. You can do this by creating a folder e.g. on the desktop.
+
+2. Open the project by clicking on :guilabel:`File` on the menu bar and then clicking on :guilabel:`Open Folder...` (see below) then navigate to your project folder.
+
+   .. image:: images/Vscode-Open_folder.png
+      :width: 600
+      :align: center
+      :alt: How to open a folder, select :guilabel:`File -> Open Folder...`
+
+3. Create a folder inside your project by clicking on the second most left icon (see below).
+
+   .. image:: images/Vscode-Create_folder.png
+      :width: 600
+      :align: center
+      :alt: How to create a folder, select :guilabel:`Explorer (Activity Bar) -> Folder+ (Icon)`
+
+Working with Files
+^^^^^^^^^^^^^^^^^^
+
+1. Create a new file inside your project by clicking on the left icon (see below) and e.g. naming it "new_file.py" - the ending ".py" makes it a python file. If you want to create a new file inside a folder in your project, make sure to click on the folder before creating the file.
+
+   .. image:: images/Vscode-files1.png
+      :width: 600
+      :align: center
+      :alt: How to create a file, select :guilabel:`Explorer (Activity Bar) -> File+ (Icon)`
+
+2. Save the file by clicking on :guilabel:`File -> Save` (see below), or by pressing :kbd:`Ctrl+S` (:kbd:`Command+S` on MacOS). If you want autosave to be enabled, click on :guilabel:`File -> Autosave`, and make sure it is checked.
+
+   .. image:: images/Vscode-Save_file.png
+      :width: 600
+      :align: center
+      :alt: How to save a file, select :guilabel:`File -> Save`, or press :kbd:`Ctrl+S` / :kbd:`Command+S` (Windows/Mac)
+
+3. To move a file to a folder simply click on the file and hold, then drag the file to the folder.
+
+
+Importing packages
+^^^^^^^^^^^^^^^^^^
```

Comment:
I guess this should be moved to a more specific "how to use python"?
We should consider where to place that one.